src/raftstore/store/peer.rs

```diff
@@ -1372,6 +1394,8 @@ impl Peer {
                 }
                 ExecResult::SplitRegion { ref left, .. } => {
                     storage.region = left.clone();
+                    self.size_diff_hint = 0;
+                    self.delete_keys_hint = 0;
```

Comment:
No, here reset `size_diff_hint` and `delete_keys_hint` to prevent further split.